eduNEXT/frontend-app-course-authoring
# perf: add support of css-variables to redwood

## Description
This PR adds compatibility of runtime theming to Redwood.

To solve the test issues it was necessary to backport this upstream fix:

- https://github.com/openedx/frontend-app-course-authoring/pull/1062

## How to test
Go to the course [authoring of the Stage](https://studio.redwood.releases.edunext.link/course-authoring/home) and navigate it. The styles should be consistent along the section and every part should work properly.

> [!NOTE]  
> The tests are failing because it is lacking a Codecov token 


[JIRA ISSUE AP-1169](https://edunext.atlassian.net/browse/AP-1169)
